plugins/vue-json-viewer.js

```diff
@@ -0,0 +1,6 @@
+import Vue from 'vue'
+import JsonViewer from 'vue-json-viewer'
+import '~/assets/scss/json-viewer.scss'
```

Comment:
probably useless
since you added your own stylesheet to assets

Reply:
Here it is needed, if I comment it out, styles are not applied